### PR TITLE
Correct hint text for times offered (not social let)

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Hint text for this question has additional hint if not a social let.

### CODE REVIEW

File: config/forms/2022_2023.json [hunk 1]
```diff
@@ -675,7 +675,7 @@
                 "offered": {
                   "check_answer_label": "Times previously offered since becoming available",
                   "header": "Since becoming available for re-let, how many times has the property been previously offered?",
-                  "hint_text": "If the property is being offered for let for the first time, enter 0.",
+                  "hint_text": "This is after the last tenancy ended. If the property is being offered for let for the first time, enter 0.",
```
Comment (roobottom) [inline config/forms/2022_2023.json:678]: Can we get Dan to OK this pls.

Comment (paulrobertlloyd) [reply config/forms/2022_2023.json:678]: @Dan13L3 Is this correct?

Comment (baarkerlounger) [reply config/forms/2022_2023.json:678]: It matches 2021_2022 so I'd expect so:

https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/config/forms/2021_2022.json#L678

**Review by baarkerlounger — APPROVED** [review]

File: config/forms/2022_2023.json (hunk 1 above)
Comment (Dan13L3) [reply config/forms/2022_2023.json:678]: @baarkerlounger @paulrobertlloyd This hint text is correct if user has selected 'no - re-let of existing social housing'

**Pull request merged**